com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -439,6 +439,21 @@ public void Blit(CommandBuffer cmd, RenderTargetIdentifier source, RenderTargetI
             cmd.Blit(source, destination, material, passIndex);
         }
 
+        /// <summary>
+        /// Add a blit command to the context for execution. This applies the material to the color target.
```

Comment:
Needs more description to tell this Blit happens with camera color target as source and dest?

Reply:
Agreed, needs to mention what it is doing behind the scenes with the swap a little bit